integration-tests/tests/block.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from 'chai';
+import SDK from "@zeitgeistpm/sdk";
```

Comment:
https://github.com/zeitgeistpm/tools/pull/105 was merged but it is cumbersome to release a new version.

@lsaether
```
npm install some_git_project#some_branch --save
```
Points to the root directory but not to `packages/sdk`. Not sure how to fetch a TS subdirectory using npm.